AutomatedTesting/Gem/PythonTests/Atom/tests/MaterialCanvas_Atom_BasicTests.py

```diff
@@ -123,7 +141,46 @@ def MaterialCanvas_BasicFunctionalityChecks_AllChecksPass():
             Tests.material_graph_name_is_test1,
             material_graph_name == "test1")
 
-        # 8. Look for errors and asserts.
+        # 8. Create a new material_graph from material_graph_document_id.
+        material_graph_document_id = material_graph_document_ids[0]
+        material_graph = material_canvas_utils.get_graph(material_graph_document_id)
+        Report.result(
+            Tests.material_graph_created,
+            material_graph.typename == "AZStd::shared_ptr<Graph>" and material_graph is not None)
+
+        # 9. Create a new world_position_node in memory.
+        world_position_node = material_canvas_utils.create_node_by_name(material_graph, "World Position")
+        Report.result(
+            Tests.world_position_node_created,
+            world_position_node.typename == "AZStd::shared_ptr<Node>" and world_position_node is not None)
+
+        # 10. Add the world_position_node to the material_document_graph.
+        # This test will be verified when the nodes are connected as if it doesn't exist the connection won't be made.
+        material_canvas_utils.add_node(material_graph_document, world_position_node, math.Vector2(-200.0, 10.0))
```

Comment:
This probably needs the graph or the graph ID, not the document ID unless you move calls to get the graph and the graph ID inside the function, which I think is fine

Reply:
now uses the graph_id returned from `atomtools.GraphDocumentRequestBus(bus.Event, GraphDocumentRequestBusEvents.GET_GRAPH_ID, graph_document_id` 

the variables are broken down as such:
`material_graph_document_id` = Uuid returned from `azlmbr.atomtools.AtomToolsDocumentSystemRequestBus(
        bus.Broadcast, AtomToolsDocumentSystemRequestBusEvents.OPEN_DOCUMENT, file_path)`
`graph_document_id` = the same as `material_graph_document_id`
`material_graph` = AZStd::shared_ptr<Graph> returned from `atomtools.GraphDocumentRequestBus(bus.Event, GraphDocumentRequestBusEvents.GET_GRAPH, graph_document_id)`
`material_graph_id` = int ID value returned from `atomtools.GraphDocumentRequestBus(bus.Event, GraphDocumentRequestBusEvents.GET_GRAPH_ID, graph_document_id)`